core/mondoo-azure-security.mql.yaml

```diff
@@ -696,7 +687,8 @@ queries:
         2. Run this query:
 
           ```bash
-          cnspec run azure -c "azure.subscription.storage.accounts { containers { properties['publicAccess'] == "None" } }"
+          cnspec run azure -c "azure.subscription.storage.accounts { properties.AllowBlobPublicAccess == "false" }"
```

Comment:
```suggestion
          cnspec run azure -c "azure.subscription.storage.accounts.all(properties.AllowBlobPublicAccess == "false" )"
```

Reply:
I intentionally left the description as-is to facilitate a better understanding for the community. This approach should help them grasp how to test effectively and enable more informed comparisons. What are your thoughts on this strategy?